Various formatting fixes in unittests

andreas-bok-sociomantic opened the pull request.

andreas-bok-sociomantic committed:
Whitespace fixes

Use google style for whitespaces
Remove double spaces and empty lines
Space around assignment operators

andreas-bok-sociomantic committed:
Format code to <=80 characters per line

andreas-bok-sociomantic committed:
Reorder imports

andreas-bok-sociomantic committed:
Beautify SQL statements

andreas-bok-sociomantic committed:
Fix indentation issues

andreas-bok-sociomantic commented:
Blocked on #106

andreas-bok-sociomantic commented:
Unblocked. The bug was a missing space and thus not related to related to #106. However this is due to the fact that the mysql version in travis-ci is v5.6. In v5.7 the current syntax is invalid.

ben-palmer-sociomantic commented on tests/unit/common.c:
```diff
@@ -58,22 +58,22 @@ void set_up_connection(void)
                       getenv("MYSQL_USER"),
                       getenv("MYSQL_PASSWORD"),
                       getenv("MYSQL_SCHEMA"), 0);
+  con = drizzle_create(getenv("MYSQL_SERVER"),
```
This change doesn't look like it should compile.

andreas-bok-sociomantic replied:
It seems like some of the changes are shown out of order/wrongly. It compiles and the location of `drizzle_create(getenv("MYSQL_SERVER")` is where it should be. 
I will try to see if it changes after force pushing and rebasing on latest v5.4.x

ben-palmer-sociomantic commented on tests/unit/connect.c:
```diff
@@ -70,11 +70,11 @@ int main(int argc, char *argv[])
   {
-    ret= drizzle_connect(con);
     ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "%s(%s)", drizzle_error(con), drizzle_strerror(ret));
+  int limit = 20;
```
Doesn't `limit` need to be defined before line 73?

andreas-bok-sociomantic replied:
It seems like some of the changes are shown out of order/wrongly. It compiles and the location of `limit` is where it should be. 
I will try to see if it changes after force pushing and rebasing on latest v5.4.x

ben-palmer-sociomantic commented on tests/unit/connect_uds.c:
```diff
@@ -67,11 +67,11 @@ int main(int argc, char *argv[])
   {
-    ret= drizzle_connect(con);
     ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "%s(%s)", drizzle_error(con), drizzle_strerror(ret));
+  int limit = 20;
```
Same here.

ben-palmer-sociomantic commented on tests/unit/insert_id.c:
```diff
@@ -57,9 +57,9 @@ int main(int argc, char *argv[])
-  drizzle_return_t ret= drizzle_connect(con);
   if (ret == DRIZZLE_RETURN_COULD_NOT_CONNECT)
   {
+  drizzle_return_t ret = drizzle_connect(con);
```
Is the indentation correct here?

ben-palmer-sociomantic commented on tests/unit/nulls.c:
```diff
@@ -75,9 +75,9 @@ int main(int argc, char *argv[])
-  drizzle_return_t driz_ret= drizzle_connect(con);
   if (driz_ret == DRIZZLE_RETURN_COULD_NOT_CONNECT)
   {
+  drizzle_return_t driz_ret = drizzle_connect(con);
```
Incorrect indentation.

ben-palmer-sociomantic commented on tests/unit/query.c:
```diff
@@ -58,9 +58,9 @@ int main(int argc, char *argv[])
-  drizzle_return_t ret= drizzle_connect(con);
   if (ret == DRIZZLE_RETURN_COULD_NOT_CONNECT)
   {
+  drizzle_return_t ret = drizzle_connect(con);
```
Incorrect indentation.

ben-palmer-sociomantic commented on tests/unit/row.c:
```diff
@@ -56,9 +56,9 @@ int main(int argc, char *argv[])
-  drizzle_return_t ret= drizzle_connect(con);
   if (ret == DRIZZLE_RETURN_COULD_NOT_CONNECT)
   {
+  drizzle_return_t ret = drizzle_connect(con);
```
Incorrect indentation.

ben-palmer-sociomantic commented on tests/unit/statement.c:
```diff
@@ -106,33 +106,33 @@ int main(int argc, char *argv[])
-    snprintf(comp_val, 3, "%"PRIu32, i);
     if (strcmp(comp_val, char_val) != 0)
     {
+    snprintf(comp_val, 3, "%" PRIu32, i);
```
This might change the functionality.

andreas-bok-sociomantic replied:
Again, something is messed up with the diff view

ben-palmer-sociomantic commented on tests/unit/column.c:
```diff
@@ -176,11 +173,10 @@ int main(int argc, char *argv[])
     ASSERT_STREQ_(drizzle_column_type_str(column_types[i]), column_type_names[i],
       "Column type 'DRIZZLE_COLUMN_TYPE_%s' resolved to wrong name: '%s'",
       column_type_names[i], drizzle_column_type_str(column_types[i]));
+  for (i = 0; i < 30; i++) {
```
This moved and will now miss the unittests.

ben-palmer-sociomantic replied:
This still applies

ben-palmer-sociomantic reviewed (commented):
Seems to me there are a few problems with this PR. There is some code moving that doesn't seem like it should be part of the PR and from what I can see should break the tests.

andreas-bok-sociomantic reviewed (commented):
Hmm, rebased and force pushed. 
Maybe it messes up because there are many changes. Promise by my mothers maiden name that it compiles all formatting issues are resolved on my computer :)

ben-palmer-sociomantic commented:
> Maybe it messes up because there are many changes. Promise by my mothers maiden name that it compiles all formatting issues are resolved on my computer :)

I just tried it on my PC and the end commit of the PR succeeds but if you just checkout the first commit (`6a1d6220e1b526f2d74d6db71992b2f324b7b464`) it fails.

andreas-bok-sociomantic commented:
> I just tried it on my PC and the end commit of the PR succeeds but if you just checkout the first commit (6a1d622) it fails.

fails as in the unittest fails or the formatting fails?

ben-palmer-sociomantic commented:
> fails as in the unittest fails or the formatting fails?

`make check` fails.

andreas-bok-sociomantic commented:
Went through the commit and fixed the bugs which caused the unittest to fail and showed the strange diff

ben-palmer-sociomantic commented on tests/unit/connect_uds.c:
```diff
@@ -67,11 +67,11 @@ int main(int argc, char *argv[])
   {
-    ret= drizzle_connect(con);
     ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "%s(%s)", drizzle_error(con), drizzle_strerror(ret));
+    ret = drizzle_connect(con);
```
Oh no! this one moved again.

ben-palmer-sociomantic commented on tests/unit/datetypes.c:
```diff
@@ -117,55 +116,55 @@ int main(int argc, char *argv[])
-  CHECK(drizzle_stmt_set_time(sth,  6, 0, 23, 59, 59, 62500, 0));
-  CHECK(drizzle_stmt_set_timestamp(sth,  7, 1642, 12, 25, 12, 15, 1, 0));
-  CHECK(drizzle_stmt_set_timestamp(sth,  8, 1642, 12, 25, 12, 12, 0, 0));
+  CHECK(drizzle_stmt_set_short(sth, 0, 4, 0));
```
Extra space in the line above this one I think.

ben-palmer-sociomantic commented on tests/unit/event_callback.c:
```diff
@@ -94,22 +94,22 @@ int main(int argc, char *argv[])
+           drizzle_strerror(driz_ret));
   ASSERT_EQ_(DRIZZLE_RETURN_OK, driz_ret, "%s(%s)", drizzle_error(con),
-    drizzle_strerror(driz_ret));
+             drizzle_strerror(driz_ret));
```
Why is this now different to the line above?

ben-palmer-sociomantic commented on tests/unit/insert_id.c:
```diff
@@ -57,9 +57,10 @@ int main(int argc, char *argv[])
+  drizzle_return_t ret = drizzle_connect(con);
   if (ret == DRIZZLE_RETURN_COULD_NOT_CONNECT)
   {
+
```
Not sure why a newline is added here.

ben-palmer-sociomantic commented on tests/unit/nulls.c:
```diff
@@ -219,8 +219,8 @@ int main(int argc, char *argv[])
       break;
-    cur_row ++;
     ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, fetching row #%d", drizzle_strerror(driz_ret), drizzle_error(con), cur_row);
+    cur_row++;
```
This line looks like it might have moved as well.

ben-palmer-sociomantic replied:
This still looks like it has moved.

ben-palmer-sociomantic commented on tests/unit/statement.c:
```diff
@@ -1,4 +1,4 @@
-/*  vim:expandtab:shiftwidth=2:tabstop=2:smarttab:
+  /*  vim:expandtab:shiftwidth=2:tabstop=2:smarttab:
```
This indentation might be a mistake.

ben-palmer-sociomantic replied:
This is still an issue.

ben-palmer-sociomantic commented on tests/unit/binlog.cc:
```diff
@@ -65,7 +67,8 @@ void binlog_event(drizzle_binlog_event_st *event, void *context) {
 }
 
-int main(int argc, char *argv[]) {
+int main(int argc, char *argv[])
```
This change seems to be the opposite of the ones done in `column.c` below.

ben-palmer-sociomantic commented on tests/unit/common.h:
```diff
@@ -60,17 +60,24 @@ extern void set_up_connection(void);
-    ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", drizzle_strerror(driz_ret), drizzle_error(con), cmd);
+/* Perform a query; assign the result to 'result' and the returncode to
+ * 'driz_ret'; assert that the returncode is DRIZZLE_RETURN_OK. */
+#define CHECKED_QUERY(cmd)                                                     \
```
I don't think all the `\`s are needed at the end of lines.

ben-palmer-sociomantic replied:
Or if they are it is syntax I haven't seen before.

ben-palmer-sociomantic reviewed (commented):
I didn't manage to quite finish this today but it seems there are still some problems with this PR.

andreas-bok-sociomantic commented:
addressed comments 

```diff
diff --git a/tests/unit/binlog.cc b/tests/unit/binlog.cc
index e987324..c95602a 100644
--- a/tests/unit/binlog.cc
+++ b/tests/unit/binlog.cc
@@ -43,15 +43,13 @@
 
 #include "tests/unit/common.h"
 
-void binlog_error(drizzle_return_t ret, drizzle_st *connection, void *context)
-{
+void binlog_error(drizzle_return_t ret, drizzle_st *connection, void *context) {
   (void)context;
   ASSERT_EQ_(DRIZZLE_RETURN_EOF, ret, "%s(%s)", drizzle_error(connection),
              drizzle_strerror(ret));
 }
 
-void binlog_event(drizzle_binlog_event_st *event, void *context)
-{
+void binlog_event(drizzle_binlog_event_st *event, void *context) {
   (void)context;
   uint32_t timestamp;
   timestamp = drizzle_binlog_event_timestamp(event);
@@ -68,8 +66,7 @@ void binlog_event(drizzle_binlog_event_st *event, void *context)
                 "Bad event type: %d", drizzle_binlog_event_type(event));
 }
 
-int main(int argc, char *argv[])
-{
+int main(int argc, char *argv[]) {
   (void)argc;
   (void)argv;
   drizzle_binlog_st *binlog;
diff --git a/tests/unit/common.h b/tests/unit/common.h
index c94c8c7..8277634 100644
--- a/tests/unit/common.h
+++ b/tests/unit/common.h
@@ -67,16 +67,16 @@ extern void tear_down_schema(const char *schema);
 
 /* Perform a query; assign the result to 'result' and the returncode to
  * 'driz_ret'; assert that the returncode is DRIZZLE_RETURN_OK. */
-#define CHECKED_QUERY(cmd)                                                     \
-  result = drizzle_query(con, cmd, 0, &driz_ret);                              \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"",       \
+#define CHECKED_QUERY(cmd) \
+  result = drizzle_query(con, cmd, 0, &driz_ret); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", \
              drizzle_strerror(driz_ret), drizzle_error(con), cmd);
 
 /* Call a libdrizzle function that just returns a drizzle_return_t. Assign the
  * return to 'driz_ret' and assert that it is DRIZZLE_RETURN_OK. */
-#define CHECK(s)                                                               \
-  driz_ret = (s);                                                              \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, in \"%s\"",         \
+#define CHECK(s) \
+  driz_ret = (s); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, in \"%s\"", \
              drizzle_strerror(driz_ret), drizzle_error(con), #s);
 
 #ifdef __cplusplus
```

ben-palmer-sociomantic commented on tests/unit/connect_uds.c:
```diff
@@ -40,12 +40,11 @@
 #include "tests/unit/common.h"
 
 #include <libdrizzle-5.1/libdrizzle.h>
+#include <stdint.h>
```
This change doesn't belong in this commit.

andreas-bok-sociomantic commented:
Addressed comments. 

```diff 
diff --git a/tests/unit/column.c b/tests/unit/column.c
index f3a885f..dd1d167 100644
--- a/tests/unit/column.c
+++ b/tests/unit/column.c
@@ -80,9 +80,8 @@ int main(int argc, char *argv[]) {
   drizzle_return_t driz_ret;
 
   set_up_connection();
-  CHECKED_QUERY(
-      "SET CHARACTER SET latin1"); /* Or any other one-byte-per-character
-                                      character set */
+  CHECKED_QUERY("SET CHARACTER SET latin1"); /* Or any other one-byte-per-character
+                                                character set */
   set_up_schema("test_column");
 
   CHECKED_QUERY("CREATE TABLE test_column.t1 (a INT PRIMARY KEY "
@@ -124,10 +123,9 @@ int main(int argc, char *argv[]) {
                    "Column max size wrong %zu != 255",
                    (size_t)drizzle_column_max_size(column));
 
-        ASSERT_EQ_(
-            drizzle_column_charset(column), DRIZZLE_CHARSET_LATIN1_SWEDISH_CI,
-            "Column charset wrong, %d != %d", drizzle_column_charset(column),
-            DRIZZLE_CHARSET_LATIN1_SWEDISH_CI);
+        ASSERT_EQ_(drizzle_column_charset(column), DRIZZLE_CHARSET_LATIN1_SWEDISH_CI,
+                   "Column charset wrong, %d != %d", drizzle_column_charset(column),
+                   DRIZZLE_CHARSET_LATIN1_SWEDISH_CI);
         break;
       case 3:
         ASSERT_EQ_(drizzle_column_type(column), DRIZZLE_COLUMN_TYPE_TIMESTAMP,
@@ -148,10 +146,9 @@ int main(int argc, char *argv[]) {
 
   /* Check that column type is resolved to the correct string representation */
   for (i = 0; i < 30; i++) {
-    ASSERT_STREQ_(
-        drizzle_column_type_str(column_types[i]), column_type_names[i],
-        "Column type 'DRIZZLE_COLUMN_TYPE_%s' resolved to wrong name: '%s'",
-        column_type_names[i], drizzle_column_type_str(column_types[i]));
+    ASSERT_STREQ_(drizzle_column_type_str(column_types[i]), column_type_names[i],
+                  "Column type 'DRIZZLE_COLUMN_TYPE_%s' resolved to wrong name: '%s'",
+                  column_type_names[i], drizzle_column_type_str(column_types[i]));
   }
 
   return EXIT_SUCCESS;
diff --git a/tests/unit/connect.c b/tests/unit/connect.c
index dee79d4..b550b04 100644
--- a/tests/unit/connect.c
+++ b/tests/unit/connect.c
@@ -46,11 +46,12 @@ int main(int argc, char *argv[]) {
   (void)argc;
   (void)argv;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"),
-      0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
diff --git a/tests/unit/connect_uds.c b/tests/unit/connect_uds.c
index c9f74c4..8283861 100644
--- a/tests/unit/connect_uds.c
+++ b/tests/unit/connect_uds.c
@@ -48,8 +48,11 @@ int main(int argc, char *argv[]) {
   (void)argc;
   (void)argv;
 
-  con = drizzle_create(getenv("MYSQL_SOCK"), 0, getenv("MYSQL_USER"),
-                       getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  con = drizzle_create(getenv("MYSQL_SOCK"),
+                       0, getenv("MYSQL_USER"),
+                       getenv("MYSQL_PASSWORD"),
+                       getenv("MYSQL_SCHEMA"), 0);
+
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
diff --git a/tests/unit/datetypes.c b/tests/unit/datetypes.c
index 0705488..61ce1bc 100644
--- a/tests/unit/datetypes.c
+++ b/tests/unit/datetypes.c
@@ -47,14 +47,14 @@
 #include <stdlib.h>
 #include <string.h>
 
-#define CHECKED_QUERY(cmd)                                                     \
-  result = drizzle_query(con, cmd, 0, &driz_ret);                              \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"",       \
+#define CHECKED_QUERY(cmd) \
+  result = drizzle_query(con, cmd, 0, &driz_ret); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", \
              drizzle_strerror(driz_ret), drizzle_error(con), cmd);
 
-#define CHECK(s)                                                               \
-  driz_ret = (s);                                                              \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, in \"%s\"",         \
+#define CHECK(s) \
+  driz_ret = (s); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, in \"%s\"", \
              drizzle_strerror(driz_ret), drizzle_error(con), #s);
 
 static const drizzle_column_type_t expected_column_types[10] = {
@@ -257,14 +257,14 @@ int main(int argc, char *argv[]) {
                drizzle_strerror(driz_ret), drizzle_error(con), 1, cur_row);
     ASSERT_EQ(cur_row, (unsigned)columnA);
 
-#define ASSERT_COL_STREQ_(coln, expected, ...)                                 \
-  col_strval = drizzle_stmt_get_string(sth, coln - 1, &lth, &driz_ret);        \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Fetching column %u of row %u",      \
-             coln, cur_row);                                                   \
-  ASSERT_STREQ_(expected, col_strval,                                          \
-                "Stringified value of column %u of row %u", coln, cur_row);    \
-  ASSERT_EQ_(lth, strlen(col_strval),                                          \
-             "Length of stringified value of column %u of row %u", coln,       \
+#define ASSERT_COL_STREQ_(coln, expected, ...) \
+  col_strval = drizzle_stmt_get_string(sth, coln - 1, &lth, &driz_ret); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Fetching column %u of row %u", \
+             coln, cur_row); \
+  ASSERT_STREQ_(expected, col_strval, \
+                "Stringified value of column %u of row %u", coln, cur_row); \
+  ASSERT_EQ_(lth, strlen(col_strval), \
+             "Length of stringified value of column %u of row %u", coln, \
              cur_row);
 
     switch (cur_row) {
@@ -364,10 +364,9 @@ int main(int argc, char *argv[]) {
       printf("row=%u col=%s: '%s' / '%s'\n", columnA, col_name, drizzle_strval,
              server_strval);
 
-      ASSERT_STREQ_(
-          server_strval, drizzle_strval,
-          "Row %u, column '%s': server strval does not match libdrizzle strval",
-          columnA, col_name);
+      ASSERT_STREQ_(server_strval, drizzle_strval,
+                    "Row %u, column '%s': server strval does not match libdrizzle strval",
+                    columnA, col_name);
     }
     ASSERT_EQ(rows_in_table, cur_row);
 
diff --git a/tests/unit/event_callback.c b/tests/unit/event_callback.c
index f252eee..f9a7b2b 100644
--- a/tests/unit/event_callback.c
+++ b/tests/unit/event_callback.c
@@ -62,7 +62,7 @@
 extern drizzle_return_t drizzle_event_callback(drizzle_st *con, short events,
                                                void *context);
 extern drizzle_return_t drizzle_event_callback(drizzle_st *con, short events,
-                                               void *context) {
+  void *context) {
   ASSERT_TRUE((events != -1));
   int *cxt_ptr = (int *)drizzle_context(con);
   ASSERT_NOT_NULL_(cxt_ptr, "Drizzle connection context is null");
@@ -83,10 +83,12 @@ int main(int argc, char *argv[]) {
   int cxt_a = 1;
   int cxt_b = 0;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   // Set drizzle dummy context
@@ -98,14 +100,14 @@ int main(int argc, char *argv[]) {
 
   drizzle_return_t driz_ret = drizzle_connect(con);
   SKIP_IF_(driz_ret == DRIZZLE_RETURN_COULD_NOT_CONNECT, "%s",
-           drizzle_strerror(driz_ret));
+    drizzle_strerror(driz_ret));
   ASSERT_EQ_(DRIZZLE_RETURN_OK, driz_ret, "%s(%s)", drizzle_error(con),
-             drizzle_strerror(driz_ret));
+    drizzle_strerror(driz_ret));
 
   driz_ret = drizzle_quit(con);
   ASSERT_EQ_(DRIZZLE_RETURN_OK, driz_ret, "%s", drizzle_strerror(driz_ret));
   ASSERT_EQ_(cxt_b > 0, true, "Unexpected number of event callbacks, Got '%d', "
-                              "Expected '1' or more", cxt_b);
+    "Expected '1' or more", cxt_b);
 
   return EXIT_SUCCESS;
 }
diff --git a/tests/unit/insert_id.c b/tests/unit/insert_id.c
index 8eae6b0..35c9bb7 100644
--- a/tests/unit/insert_id.c
+++ b/tests/unit/insert_id.c
@@ -43,20 +43,21 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#define CHECKED_QUERY(cmd)                                                     \
-  drizzle_query(con, cmd, 0, &ret);                                            \
-  ASSERT_EQ_(ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"",            \
+#define CHECKED_QUERY(cmd) \
+  drizzle_query(con, cmd, 0, &ret); \
+  ASSERT_EQ_(ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", \
              drizzle_strerror(ret), drizzle_error(con), cmd);
 
 int main(int argc, char *argv[]) {
   (void)argc;
   (void)argv;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"),
-      0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
@@ -77,9 +78,8 @@ int main(int argc, char *argv[]) {
   ret = drizzle_select_db(con, "test_insert");
   ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "USE test_insert");
 
-  CHECKED_QUERY(
-      "create table test_insert.t1 ( a int primary key auto_increment,"
-      "b int)");
+  CHECKED_QUERY("create table test_insert.t1 ( a int primary key auto_increment,"
+                "b int)");
 
   drizzle_result_st *result = drizzle_query(
       con, "insert into test_insert.t1 (b) values (1),(2),(3)", 0, &ret);
diff --git a/tests/unit/nulls.c b/tests/unit/nulls.c
index 58c5929..1e08f36 100644
--- a/tests/unit/nulls.c
+++ b/tests/unit/nulls.c
@@ -53,9 +53,9 @@
 #include <stdlib.h>
 #include <string.h>
 
-#define CHECKED_QUERY(cmd)                                                     \
-  result = drizzle_query(con, cmd, 0, &driz_ret);                              \
-  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"",       \
+#define CHECKED_QUERY(cmd) \
+  result = drizzle_query(con, cmd, 0, &driz_ret); \
+  ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", \
              drizzle_strerror(driz_ret), drizzle_error(con), cmd);
 
 #define TEST_PREPARED_STATEMENTS
@@ -71,11 +71,12 @@ int main(int argc, char *argv[]) {
   drizzle_stmt_st *sth;
 #endif
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"),
-      0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t driz_ret = drizzle_connect(con);
@@ -108,12 +109,12 @@ int main(int argc, char *argv[]) {
   for (int sym = 0; sym < 64; sym++) {
     int cn = 0;
     *p++ = '(';
-#define APPENDMAYBE(b)                                                         \
-  if (sym & b) {                                                               \
-    strcpy(p, "NULL");                                                         \
-    p += 4;                                                                    \
-  } else {                                                                     \
-    p += sprintf(p, "%d", sym * NCOLS + cn);                                   \
+#define APPENDMAYBE(b) \
+  if (sym & b) { \
+    strcpy(p, "NULL"); \
+    p += 4; \
+  } else { \
+    p += sprintf(p, "%d", sym * NCOLS + cn); \
   }
     APPENDMAYBE(0x01);
     *p++ = ',';
@@ -153,12 +154,12 @@ int main(int argc, char *argv[]) {
 // ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, resetting
 // statement", drizzle_strerror(driz_ret), drizzle_error(con));
 
-#define SETMAYBE(cn, b)                                                        \
-  if (sym & b)                                                                 \
-    driz_ret = drizzle_stmt_set_null(sth, cn);                                 \
-  else                                                                         \
+#define SETMAYBE(cn, b) \
+  if (sym & b) \
+    driz_ret = drizzle_stmt_set_null(sth, cn); \
+  else \
     SETALWAYS(cn);
-#define SETALWAYS(cn)                                                          \
+#define SETALWAYS(cn) \
   driz_ret = drizzle_stmt_set_short(sth, cn, sym * NCOLS + cn + 1000, 0)
     ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK);
     SETMAYBE(0, 0x01);
@@ -219,15 +220,15 @@ int main(int argc, char *argv[]) {
       rowbase = 1000 + sym * NCOLS;
     }
 
-#define NULLMAYBE(cn, b)                                                       \
-  if (sym & b) {                                                               \
-    ASSERT_NULL_(row[cn], "Column %d, row %d should be NULL", cn + 1,          \
-                 cur_row);                                                     \
-  } else {                                                                     \
-    ASSERT_NOT_NULL_(row[cn], "Column %d, row %d should not be NULL", cn + 1,  \
-                     cur_row);                                                 \
-    sprintf(nbuf, "%d", rowbase + cn);                                         \
-    ASSERT_STREQ(row[cn], nbuf);                                               \
+#define NULLMAYBE(cn, b) \
+  if (sym & b) { \
+    ASSERT_NULL_(row[cn], "Column %d, row %d should be NULL", cn + 1, \
+                 cur_row); \
+  } else { \
+    ASSERT_NOT_NULL_(row[cn], "Column %d, row %d should not be NULL", cn + 1, \
+                     cur_row); \
+    sprintf(nbuf, "%d", rowbase + cn); \
+    ASSERT_STREQ(row[cn], nbuf); \
   }
 #define NULLNEVER(cn) NULLMAYBE(cn, 0)
 
@@ -287,26 +288,26 @@ int main(int argc, char *argv[]) {
     bool isNull;
     uint32_t rowvalue;
 
-#define GETNULLNESS(cn)                                                        \
-  isNull = drizzle_stmt_get_is_null(sth, cn, &driz_ret);                       \
+#define GETNULLNESS(cn) \
+  isNull = drizzle_stmt_get_is_null(sth, cn, &driz_ret); \
   ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK);
-#define NULLNOTNOW(cn)                                                         \
-  ASSERT_FALSE_(isNull, "Column %d, row %d should not be NULL", cn + 1,        \
-                cur_row);                                                      \
-  rowvalue = drizzle_stmt_get_int(sth, cn, &driz_ret);                         \
-  ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK);                                      \
-  ASSERT_EQ_(rowvalue, (unsigned)(rowbase + cn),                               \
-             "Column %d, row %d has unexpected data, expected: %d, got: %d",   \
+#define NULLNOTNOW(cn) \
+  ASSERT_FALSE_(isNull, "Column %d, row %d should not be NULL", cn + 1, \
+                cur_row); \
+  rowvalue = drizzle_stmt_get_int(sth, cn, &driz_ret); \
+  ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK); \
+  ASSERT_EQ_(rowvalue, (unsigned)(rowbase + cn), \
+             "Column %d, row %d has unexpected data, expected: %d, got: %d", \
              cn + 1, cur_row, rowbase + cn, rowvalue);
-#define NULLMAYBE(cn, b)                                                       \
-  GETNULLNESS(cn);                                                             \
-  if (sym & b) {                                                               \
+#define NULLMAYBE(cn, b) \
+  GETNULLNESS(cn); \
+  if (sym & b) { \
     ASSERT_TRUE_(isNull, "Column %d, row %d should be NULL", cn + 1, cur_row); \
-  } else {                                                                     \
-    NULLNOTNOW(cn);                                                            \
+  } else { \
+    NULLNOTNOW(cn); \
   }
-#define NULLNEVER(cn)                                                          \
-  GETNULLNESS(cn);                                                             \
+#define NULLNEVER(cn) \
+  GETNULLNESS(cn); \
   NULLNOTNOW(cn);
 
     NULLMAYBE(0, 0x01);
diff --git a/tests/unit/numbers.c b/tests/unit/numbers.c
index 06675b9..ab87de9 100644
--- a/tests/unit/numbers.c
+++ b/tests/unit/numbers.c
@@ -50,12 +50,12 @@
  * binary floating-point format, so we should get exact results out. However,
  * some compilers (pretty reasonably) warn when doing exact equality tests of
  * floats. This gets around that. */
-#define FLOAT_EQ_EXACT(a, b)                                                   \
+#define FLOAT_EQ_EXACT(a, b) \
   (isfinite(a) && isfinite(b) && !(a < b) && !(a > b))
 
-#define ASSERT_FLOATEQEXACT(a, b)                                              \
-  ASSERT_TRUE_(FLOAT_EQ_EXACT(a, b),                                           \
-               "Floating-point values %f != %f (difference is %g)", (a), (b),  \
+#define ASSERT_FLOATEQEXACT(a, b) \
+  ASSERT_TRUE_(FLOAT_EQ_EXACT(a, b), \
+               "Floating-point values %f != %f (difference is %g)", (a), (b), \
                (a) - (b))
 
 static const drizzle_column_type_t expected_column_types[9] = {
@@ -248,11 +248,9 @@ int main(int argc, char *argv[]) {
 
       /* These columns are 0-based */
       col_val = drizzle_stmt_get_bigint(sth, cur_column - 1, &driz_ret);
-      ASSERT_EQ_(
-          driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
-          drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
-      col_strval =
-          drizzle_stmt_get_string(sth, cur_column - 1, &lth, &driz_ret);
+      ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
+                 drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
+      col_strval = drizzle_stmt_get_string(sth, cur_column - 1, &lth, &driz_ret);
       ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK);
       ASSERT_EQ(lth, strlen(col_strval));
       col_dblval = drizzle_stmt_get_double(sth, cur_column - 1, &driz_ret);
@@ -321,15 +319,14 @@ int main(int argc, char *argv[]) {
     ASSERT_FLOATEQEXACT((float)col_val, trunc(expect_floatval));
 
     col_dblval = drizzle_stmt_get_double(sth, 6, &driz_ret);
-    ASSERT_EQ_(
-        driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
-        drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
-    ASSERT_FLOATEQEXACT(col_dblval, expect_floatval);
+    ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
+               drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
+               ASSERT_FLOATEQEXACT(col_dblval, expect_floatval);
 
     col_strval = drizzle_stmt_get_string(sth, 6, &lth, &driz_ret);
     ASSERT_EQ(driz_ret, DRIZZLE_RETURN_OK);
 
-    printf("  Column %d: %" PRIdMAX "  \"%s\"   %f\n", 7, col_val, col_strval,
+    printf("  Column %d: %" PRIdMAX " \"%s\" %f\n", 7, col_val, col_strval,
            col_dblval);
 
     col_val = drizzle_stmt_get_int(sth, 7, &driz_ret);
@@ -347,9 +344,8 @@ int main(int argc, char *argv[]) {
     ASSERT_FLOATEQEXACT((double)col_val, trunc(expect_dblval));
 
     col_dblval = drizzle_stmt_get_double(sth, 7, &driz_ret);
-    ASSERT_EQ_(
-        driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
-        drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
+    ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, column %d of row %d",
+               drizzle_strerror(driz_ret), drizzle_error(con), cur_column, cur_row);
     ASSERT_FLOATEQEXACT(col_dblval, expect_dblval);
 
     col_strval = drizzle_stmt_get_string(sth, 7, &lth, &driz_ret);
diff --git a/tests/unit/query.c b/tests/unit/query.c
index 06e910d..73579d9 100644
--- a/tests/unit/query.c
+++ b/tests/unit/query.c
@@ -42,9 +42,9 @@
 #include <stdlib.h>
 #include <string.h>
 
-#define CHECKED_QUERY(cmd)                                                     \
-  drizzle_query(con, cmd, 0, &ret);                                            \
-  ASSERT_EQ_(ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"",            \
+#define CHECKED_QUERY(cmd) \
+  drizzle_query(con, cmd, 0, &ret); \
+  ASSERT_EQ_(ret, DRIZZLE_RETURN_OK, "Error (%s): %s, from \"%s\"", \
              drizzle_strerror(ret), drizzle_error(con), cmd);
 
 int main(int argc, char *argv[]) {
@@ -53,10 +53,12 @@ int main(int argc, char *argv[]) {
   drizzle_row_t row;
   int num_fields;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
diff --git a/tests/unit/row.c b/tests/unit/row.c
index fc01b37..ea73df9 100644
--- a/tests/unit/row.c
+++ b/tests/unit/row.c
@@ -48,11 +48,12 @@ int main(int argc, char *argv[]) {
   drizzle_row_t row;
   int num_fields;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"),
-      0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
@@ -89,8 +90,7 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
-  drizzle_result_st *result =
-      drizzle_query(con, "select * from test_row.t1", 0, &ret);
+  drizzle_result_st *result = drizzle_query(con, "select * from test_row.t1", 0, &ret);
   if (ret != DRIZZLE_RETURN_OK) {
     printf("Select failure\n");
     return EXIT_FAILURE;
diff --git a/tests/unit/statement.c b/tests/unit/statement.c
index dfb0818..705bce1 100644
--- a/tests/unit/statement.c
+++ b/tests/unit/statement.c
@@ -52,10 +52,12 @@ int main(int argc, char *argv[]) {
 
   ASSERT_EQ(UINT64_MAX, drizzle_stmt_row_count(stmt));
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);
diff --git a/tests/unit/statement_char.c b/tests/unit/statement_char.c
index b979a02..182ea92 100644
--- a/tests/unit/statement_char.c
+++ b/tests/unit/statement_char.c
@@ -50,10 +50,12 @@ int main(int argc, char *argv[]) {
   (void)argv;
   drizzle_stmt_st *stmt;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
 
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
@@ -84,10 +86,9 @@ int main(int argc, char *argv[]) {
   ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "CREATE TABLE test_stmt_ch.t1 (a INT): %s",
              drizzle_error(con));
 
-  drizzle_query(
-      con,
-      "INSERT INTO test_stmt_ch.t1 VALUES ('hello'),('drizzle'),('people')", 0,
-      &ret);
+  drizzle_query(con,
+	        "INSERT INTO test_stmt_ch.t1 VALUES ('hello'),('drizzle'),('people')", 
+		0, &ret);
   ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "%s", drizzle_error(con));
 
   const char *query = "SELECT * FROM test_stmt_ch.t1 WHERE a = ?";
diff --git a/tests/unit/unbuffered_query.c b/tests/unit/unbuffered_query.c
index 26e9ec5..92650c5 100644
--- a/tests/unit/unbuffered_query.c
+++ b/tests/unit/unbuffered_query.c
@@ -48,10 +48,12 @@ int main(int argc, char *argv[]) {
   drizzle_row_t row;
   int num_fields;
 
-  drizzle_st *con = drizzle_create(
-      getenv("MYSQL_SERVER"),
-      getenv("MYSQL_PORT") ? atoi("MYSQL_PORT") : DRIZZLE_DEFAULT_TCP_PORT,
-      getenv("MYSQL_USER"), getenv("MYSQL_PASSWORD"), getenv("MYSQL_SCHEMA"), 0);
+  drizzle_st *con = drizzle_create(getenv("MYSQL_SERVER"),
+                                   getenv("MYSQL_PORT") ? atoi("MYSQL_PORT")
+                                                        : DRIZZLE_DEFAULT_TCP_PORT,
+                                   getenv("MYSQL_USER"),
+                                   getenv("MYSQL_PASSWORD"),
+                                   getenv("MYSQL_SCHEMA"), 0);
   ASSERT_NOT_NULL_(con, "Drizzle connection object creation error");
 
   drizzle_return_t ret = drizzle_connect(con);

```

ben-palmer-sociomantic commented on tests/unit/numbers.c:
```diff
@@ -90,20 +99,27 @@ int main(int argc, char *argv[]) {
 
-  CHECKED_QUERY("create table test_numbers.t1 (a int primary key auto_increment, b tinyint, c smallint, d mediumint, e int, f bigint, g float, h double(16,13))");
+  CHECKED_QUERY("create table test_numbers.t1 (a int primary key "
+                "auto_increment, b tinyint, c smallint, d mediumint, e int, f "
```
Its a pretty weird break location to break before the `auto_increment`.

ben-palmer-sociomantic commented on tests/unit/numbers.c:
```diff
@@ -90,20 +99,27 @@ int main(int argc, char *argv[]) {
+  CHECKED_QUERY("insert into test_numbers.t1 (b,c,d,e,f,g,h) values "
+                "(1,1,1,1,1,1,1), "
+                "(127,32687,8388351,2147352575,9222246136947920895,443664,291."
+                "2711110711098);");
```
This is also a pretty strange break location between `291.` and `2711110711098`

andreas-bok-sociomantic replied:
Ah upper case is back :)

ben-palmer-sociomantic commented on tests/unit/statement.c:
```diff
@@ -65,27 +65,32 @@ int main(int argc, char *argv[]) {
 
   ret = drizzle_select_db(con, "test_stmt");
-  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "USE test_stmt");
+  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "use test_stmt");
```
Any reason `use` is changed here?

ben-palmer-sociomantic commented on tests/unit/statement.c:
```diff
@@ -80,16 +80,16 @@ int main(int argc, char *argv[]) {
 
   ret = drizzle_select_db(con, "test_stmt");
-  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "use test_stmt");
+  ASSERT_EQ_(DRIZZLE_RETURN_OK, ret, "USE test_stmt");
```
and its back :)

andreas-bok-sociomantic replied:
But shouldn't this be correct, the **ASSERT_EQ** is after the call to `drizzle_select_db`?

ben-palmer-sociomantic reviewed (commented):
I had a few comments then LGTM once the curly bracket formatting commit is removed.

andreas-bok-sociomantic commented:
Fix wrong change, https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/108#discussion-diff-105138714R83

andreas-bok-sociomantic commented:
Went over the commits again and fixed wrong changes 

```diff
diff --git a/tests/unit/binlog.cc b/tests/unit/binlog.cc
index c95602a..e987324 100644
--- a/tests/unit/binlog.cc
+++ b/tests/unit/binlog.cc
@@ -43,13 +43,15 @@
 
 #include "tests/unit/common.h"
 
-void binlog_error(drizzle_return_t ret, drizzle_st *connection, void *context) {
+void binlog_error(drizzle_return_t ret, drizzle_st *connection, void *context)
+{
   (void)context;
   ASSERT_EQ_(DRIZZLE_RETURN_EOF, ret, "%s(%s)", drizzle_error(connection),
              drizzle_strerror(ret));
 }
 
-void binlog_event(drizzle_binlog_event_st *event, void *context) {
+void binlog_event(drizzle_binlog_event_st *event, void *context)
+{
   (void)context;
   uint32_t timestamp;
   timestamp = drizzle_binlog_event_timestamp(event);
@@ -66,7 +68,8 @@ void binlog_event(drizzle_binlog_event_st *event, void *context) {
                 "Bad event type: %d", drizzle_binlog_event_type(event));
 }
 
-int main(int argc, char *argv[]) {
+int main(int argc, char *argv[])
+{
   (void)argc;
   (void)argv;
   drizzle_binlog_st *binlog;
diff --git a/tests/unit/column.c b/tests/unit/column.c
index 5934698..be9d4a2 100644
--- a/tests/unit/column.c
+++ b/tests/unit/column.c
@@ -85,12 +85,12 @@ int main(int argc, char *argv[])
                                                 character set */
   set_up_schema("test_column");
 
-  CHECKED_QUERY("CREATE TABLE test_column.t1 (a INT PRIMARY KEY "
-                "AUTO_INCREMENT, b VARCHAR(255), c TIMESTAMP DEFAULT "
-                "CURRENT_TIMESTAMP)");
+  CHECKED_QUERY("create table test_column.t1 (a int primary key auto_increment, "
+                "b varchar(255), c timestamp default current_timestamp)");
+  CHECKED_QUERY("CREATE TABLE test_column.t1 (a INT PRIMARY KEY AUTO_INCREMENT, "
+		"b VARCHAR(255), c TIMESTAMP DEFAULT CURRENT_TIMESTAMP)");
 
-  CHECKED_QUERY(
-      "INSERT INTO test_column.t1 (b) VALUES ('this'),('is'),('war')");
+  CHECKED_QUERY("INSERT INTO test_column.t1 (b) VALUES ('this'),('is'),('war')");
 
   CHECKED_QUERY("SELECT * FROM test_column.t1");
 
diff --git a/tests/unit/datetypes.c b/tests/unit/datetypes.c
index 393d6a0..c77f241 100644
--- a/tests/unit/datetypes.c
+++ b/tests/unit/datetypes.c
@@ -360,8 +360,7 @@ int main(int argc, char *argv[])
                  col_name, columnA);
       ASSERT_EQ(strlen(drizzle_strval), drizzle_strval_lth);
 
-      server_strval =
-          drizzle_stmt_get_string(sth, 2, &server_strval_lth, &driz_ret);
+      server_strval = drizzle_stmt_get_string(sth, 2, &server_strval_lth, &driz_ret);
       ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Cast column '%s' of row %u",
                  col_name, columnA);
       ASSERT_EQ(strlen(server_strval), server_strval_lth);
diff --git a/tests/unit/nulls.c b/tests/unit/nulls.c
index 0eca3cd..699d418 100644
--- a/tests/unit/nulls.c
+++ b/tests/unit/nulls.c
@@ -254,8 +254,7 @@ int main(int argc, char *argv[])
   drizzle_result_free(result);
 
 #ifdef TEST_PREPARED_STATEMENTS
-  strcpy(querybuf,
-         "SELECT a,b,c,d,e,f,g,h,i,j,k FROM test_nulls.t1 ORDER BY e");
+  strcpy(querybuf, "SELECT a,b,c,d,e,f,g,h,i,j,k FROM test_nulls.t1 ORDER BY e");
   sth = drizzle_stmt_prepare(con, querybuf, strlen(querybuf), &driz_ret);
   ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK, "Error (%s): %s, preparing \"%s\"",
              drizzle_strerror(driz_ret), drizzle_error(con), querybuf);
diff --git a/tests/unit/numbers.c b/tests/unit/numbers.c
index 8b5710d..56bf4ba 100644
--- a/tests/unit/numbers.c
+++ b/tests/unit/numbers.c
@@ -272,8 +272,7 @@ int main(int argc, char *argv[])
                  "Column %d of row %d: expected %" PRIdMAX ", got %" PRIdMAX,
                  cur_column, cur_row, expect_val, col_val);
       if (cur_column <= 5) {
-        uint32_t col_narrowval =
-            drizzle_stmt_get_int(sth, cur_column - 1, &driz_ret);
+        uint32_t col_narrowval = drizzle_stmt_get_int(sth, cur_column - 1, &driz_ret);
         ASSERT_EQ_(driz_ret, DRIZZLE_RETURN_OK,
                    "Error (%s): %s, column %d of row %d",
                    drizzle_strerror(driz_ret), drizzle_error(con), cur_column,
diff --git a/tests/unit/query.c b/tests/unit/query.c
index f08dbfc..ef7730a 100644
--- a/tests/unit/query.c
+++ b/tests/unit/query.c
@@ -95,8 +95,7 @@ int main(int argc, char *argv[])
     return EXIT_FAILURE;
   }
 
-  drizzle_result_st *result =
-      drizzle_query(con, "select * from test_query.t1", 0, &ret);
+  drizzle_result_st *result = drizzle_query(con, "select * from test_query.t1", 0, &ret);
   if (ret != DRIZZLE_RETURN_OK)
   {
     printf("Select failure\n");
diff --git a/tests/unit/row.c b/tests/unit/row.c
index 51a90c8..bc25951 100644
--- a/tests/unit/row.c
+++ b/tests/unit/row.c
@@ -95,7 +95,8 @@ int main(int argc, char *argv[])
   }
 
   drizzle_result_st *result = drizzle_query(con, "SELECT * FROM test_row.t1", 0, &ret);
-  if (ret != DRIZZLE_RETURN_OK) {
+  if (ret != DRIZZLE_RETURN_OK)
+  {
     printf("Select failure\n");
     return EXIT_FAILURE;
   }

```

ben-palmer-sociomantic commented:
LGTM
